### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by 0xTex**
Add period to end of paragraph

### CODE REVIEW

File: docs/S01-fundamentals/M0-intro/index.html [hunk 1]
```diff
@@ -74,7 +74,9 @@ <h3>What's the Proposed Solution?</h3>
 
   <h2>What Makes Up a Protocol?</h2>
 
-  <p>In the following modules, we're going to examine the elements engineers have used to build the peer-to-peer blockchain protocol. We'll then see how systems like Bitcoin and Ethereum join these separate elements together to make a wholly new product: blockchain. Last, we'll talk about the development of Bitcoin and how you can start playing with the Ethereum blockchain right from your browser.</p>
+  <p>
+    In the following modules, we're going to examine the elements engineers have used to build the peer-to-peer blockchain protocol. We'll then see how systems like Bitcoin and Ethereum join these separate elements together to make a wholly new product: blockchain. Last, we'll talk about the development of Bitcoin and how you can start playing with the Ethereum blockchain right from your browser.
```
Comment (ConsenSys-Academy) [inline docs/S01-fundamentals/M0-intro/index.html:78]: Hi! Could you correct this so the <p> is right next to the text? Will merge when you do that!

**Comment by cooganb** [comment]
Hey! Lmk if you'd like to try and fix this, otherwise I'll close in 24hrs

**Comment by 0xTex** [comment]
Resolved line 77 & 78 conflict

**Comment by ConsenSys-Academy** [comment]
Awesome! Thanks

**Pull request merged**